packages/react-components/react-nav-preview/library/src/components/sharedNavStyles.styles.ts

```diff
@@ -32,6 +37,11 @@ export const useRootDefaultClassName = makeResetStyles({
   // this element can change between a button and an anchor
   // so we need to reset box sizing to prevent horizontal overflow
   boxSizing: 'border-box',
+
+  transitionDuration: tokens.durationFast,
```

Comment:
Can you explain what these are needed for? Looks like `animationTokens` should handle all the animations?

Reply:
Transitions seemed like the simpler option since the indicator is just animating in. Transitions were simply fewer lines to write and read.

I felt like I needed more nuanced control over the icons, so that's why I went two separate routes

🙂

Reply:
ah - did you mean using the tokens nav animation tokens I created? Sure - I can apply those :-)